templates/census-thank-you.html

```diff
@@ -48,10 +48,12 @@ <h1 data-qa="title">
     <p>{{ _("Your personal information is protected by law and will be kept confidential") }}</p>
   {% endcall %}
 
-  {% if form %}
-    <h2>{{ _("Get confirmation email") }}</h2>
-    <p>{{ _("If you would like to be sent confirmation that you have completed your census enter your email address") }}</p>
-    {% include 'partials/email-form.html' %}
+  {% if content.show_confirmation_email_form %}
+    {% if form %}
+      <h2>{{ _("Get confirmation email") }}</h2>
+      <p>{{ _("If you would like to be sent confirmation that you have completed your census enter your email address") }}</p>
+      {% include 'partials/email-form.html' %}
+    {% endif %}
```

Comment:
The `show_confirmation_email_form` conditional isn't needed now that form is only present when the limit hasn't been reached.
```suggestion
{% if form %}
  <h2>{{ _("Get confirmation email") }}</h2>
  <p>{{ _("If you would like to be sent confirmation that you have completed your census enter your email address") }}</p>
  {% include 'partials/email-form.html' %}
```